### PR TITLE
Reducing hash calls

### PR DESCRIPTION
**Pull request by Whattabatt**

### CODE REVIEW

File: src/shard.rs [hunk 1]
```diff
@@ -601,6 +601,7 @@ impl FileCache {
                 &path,
                 Some(3), // retry twice if fail
             ))?;
+            log::info!("Download complete.");
```
Comment (undfined) [inline src/shard.rs:604]: Still needed?

File: pyproject.toml [hunk 2]
```diff
@@ -8,7 +8,7 @@ requires-python = ">=3.8"
 dependencies = [
     "anyascii>=0.3.2",
     "blingfire==0.1.8",
-    "boto3>=1.28",
+    "boto3[crt]>=1.28",
```
Comment (undfined) [inline pyproject.toml:11]: Since we can't use this on most instance types let's drop it for now.

File: Cargo.toml [hunk 3]
```diff
@@ -3,6 +3,8 @@ name = "dolma"
 version = "1.0.3"
 edition = "2021"
 license = "Apache-2.0"
+inherits = "release"
+debug = true
```
Comment (undfined) [inline Cargo.toml:7]: Should this be default?

**Review by undfined — APPROVED** [review]
Looks good with some questions... but post those gainz :)

File: src/shard.rs (hunk 1 above)
Comment (Whattabatt) [reply src/shard.rs:604]: Yeah, it's the only clear signal that the download is finished

Comment (soldni) [reply src/shard.rs:604]: Can we maybe have file path too?

File: pyproject.toml (hunk 2 above)
Comment (soldni) [reply pyproject.toml]: we can make it optional

**Pull request merged**